contracts/azorius/Azorius.sol

```diff
@@ -318,6 +314,11 @@ contract Azorius is Module, IAzorius {
             return ProposalState.ACTIVE;
         } else if (!_strategy.isPassed(_proposalId)) {
             return ProposalState.FAILED;
+        } else if (_proposal.txHashes.length == 0) {
+            // a Proposal with 0 transactions goes straight to EXECUTED
+            // this allows for the potential for on-chain voting for 
+            // "off-chain" executed decisions
+            return ProposalState.EXECUTED;
         } else if (
             block.number <= votingEndBlock + _proposal.timelockPeriod
         ) {
```

Comment:
I think its unnecessary to add an additional else block for this special case, if we move the original 'EXECUTED' block up above the 'TIMELOCKED' block, it will catch this special case. So could be simplified to:
![image](https://user-images.githubusercontent.com/84364476/225777682-380e9097-28c2-4558-b938-8b89ac08613f.png)